shared/libraries/opcuatms/opcuatms_client/src/objects/tms_client_property_impl.cpp

```diff
@@ -161,20 +161,17 @@ void TmsClientPropertyImpl::configurePropertyFields()
                             // devices working which are deliviering not a default value via the opc-ua interface.
                             // Afterwards, the workaround needs to be rolled back.
 
-                            try
+                            auto value = reader->getValue(childNodeId, UA_ATTRIBUTEID_VALUE);
+                            if(value.isNull())
                             {
-                                const auto value = reader->getValue(childNodeId, UA_ATTRIBUTEID_VALUE);
-                                this->defaultValue = VariantConverter<IBaseObject>::ToDaqObject(value, daqContext);
-                            }
-                            catch (const std::exception& e)
-                            {
-                                const auto value = reader->getValue(nodeId, UA_ATTRIBUTEID_VALUE);
+                                value = reader->getValue(nodeId, UA_ATTRIBUTEID_VALUE);
                                 this->defaultValue = VariantConverter<IBaseObject>::ToDaqObject(value, daqContext);
                                 LOG_W(
-                                    "Failed to read default value of property {} on OpcUa client. Detault value is set to the value at connection time. {}",
-                                    this->name,
-                                    e.what());
+                                    "Failed to read default value of property {} on OpcUa client. Detault value is set to the value at connection time.",
+                                    this->name);
                             }
+                            this->defaultValue = VariantConverter<IBaseObject>::ToDaqObject(value, daqContext);
+                            
```

Comment:
This should still be wrapped in a try/catch, even if the case for isNull is handled.

Reply:
Disagree - because then also all other statements in this file need a try catch block. So, adding then here a try catch block is inconsistent in my point of view.